test/acceptance/admin_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'acceptance/test_helper'
+
+class AdminTest < AcceptanceTest
+  test 'Admin should be able to manually run suite for existing repo' do
+    repo = create(:repo)
+
+    visit admin_repo_path(repo.name)
+    page.must_have_content 'Run last'
```

Comment:
Using `i18n` for text that are displayed will also allow you to avoid duplications in the test cases.